i3lock.c

```diff
@@ -802,14 +834,15 @@ int main(int argc, char *argv[]) {
         {"ignore-empty-password", no_argument, NULL, 'e'},
         {"inactivity-timeout", required_argument, NULL, 'I'},
         {"show-failed-attempts", no_argument, NULL, 'f'},
+        {"crash-handler", required_argument, NULL, 'C'},
```

Comment:
I don't think we need a one letter parameter for this. C in particular doesn't really speak to me. Plus it has a particular meaning for other tools from the i3 suite. I think the flag should communicate more clearly what format the argument has (a command). Something like `--command-on-crash`?